tests/scheduled_checks.py

```diff
@@ -3,67 +3,140 @@
 
-DB_NAME = "temp.sqlite"
+DB_NAME = "temp_SIMPLE.sqlite"
 DB_PATH = "data"
 
 
-# Load the database for use in individual tests
+# Load the SIMPLE database
 @pytest.fixture(scope="module")
 def db():
-    # Create a fresh temporary database and assert it exists
-    # Because we've imported simple.schema, we will be using that schema for the database
-
-    if os.path.exists(DB_NAME):
-        os.remove(DB_NAME)
-    connection_string = "sqlite:///" + DB_NAME
-    create_database(connection_string)
-    assert os.path.exists(DB_NAME)
 
     # Connect to the new database and confirm it has the Sources table
-    db = Database(connection_string, reference_tables=REFERENCE_TABLES)
-    assert db
-    assert "source" in [c.name for c in db.Sources.columns]
-
-    # Load data into an in-memory sqlite database first, for performance
-
-    # create and connects to a temporary in-memory database
-    temp_db = Database("sqlite://", reference_tables=REFERENCE_TABLES)
+    db = load_astrodb(DB_NAME, data_path=DB_PATH, reference_tables=REFERENCE_TABLES)
 
-    # load the data from the data files into the database
-    temp_db.load_database(DB_PATH, verbose=False)
+    return db
 
-    # dump in-memory database to file
-    temp_db.dump_sqlite(DB_NAME)
-    # replace database object with new file version
-    db = Database("sqlite:///" + DB_NAME, reference_tables=REFERENCE_TABLES)
 
-    return db
+def test_db(db):
+    assert os.path.exists(DB_NAME)
+    assert db
+    assert "source" in [c.name for c in db.Sources.columns]
 
 
 def test_spectra_urls(db):
     spectra_urls = db.query(db.Spectra.c.access_url).astropy()
     broken_urls = []
     codes = []
-    internet, _ = internet_connection()
-    if internet:
-        for spectrum_url in tqdm(spectra_urls["access_url"]):
-            request_response = requests.head(spectrum_url)
-            status_code = request_response.status_code
-            # The website is up if the status code is 200
-            # cuny academic commons links give 301 status code
-            if status_code != 200 and status_code != 301:
-                broken_urls.append(spectrum_url)
-                codes.append(status_code)
+    # internet, _ = internet_connection()
+    # if not internet:
+    #    assert False, "No internet connection to check spectra urls"
+
+    for spectrum_url in tqdm(spectra_urls["access_url"]):
+        request_response = requests.head(spectrum_url)
+        status_code = request_response.status_code
+        # The website is up if the status code is 200
+        # cuny academic commons links give 301 status code
+        if status_code != 200 and status_code != 301:
+            broken_urls.append(spectrum_url)
+            codes.append(status_code)
 
     # Display broken spectra regardless if it's the number we expect or not
     print(f"found {len(broken_urls)} broken spectra urls: {broken_urls}, {codes}")
 
-    assert 4 <= len(broken_urls) <= 4
+    assert 5 <= len(broken_urls) <= 5
```

Comment:
Not sure what this means? You want to ensure you have less than or equal to 5 broken urls but also greater than or equal to 5? Do you need both the greater and the less than?